com.seafile.Client.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: com.seafile.Client
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: seafile-applet
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --persist=.ccnet
+  - --filesystem=home/Seafile:create # Where user libraries are stored by default
```

Comment:
Does that actually work? I've seen `~/Seafile` in the past, not `home`.

Reply:
I'm not sure what you mean. `--filesystem=home/Seafile` means that the application has access to `~/Seafile`, doesn't it?

Reply:
@barthalion I now have a Seafile folder in my home directory so it does seem to!